README.md

```diff
@@ -133,6 +133,8 @@ We use git tags to tag releases and github's release feature to deploy.  The ste
      ('v0.12.0')
   9. Add a quick 1-2 sentence summary, make sure 'Set as the latest release' is enabled and hit 'Publish release'
   10. Check the repo's `Actions` tab to observe the progress of the deployment to production
+    - Note that the deployment job merely kicks off an ECS service update. To fully verify success, you'll need to check the
+      `Deployments` tab for the relevant service / cluster in the ECS console.
```

Comment:
The addition of this blurb helps with clarifying the deployment process more since I didn't even know about this part either.